### DISCUSSION TITLE
Beginners question..

### DISCUSSION THREAD

**Discussion by renrom** [Budibase/budibase#3961] [Help] [answered]
Hi, 

I am very new to this and i am curious if this is possible with budibase:

Table 1:
PartNumber
Brand
Date 

Table 2:
FirstName
LastName
Date

The ultimate goal is to create a form (so i need another table with links to table 1 and 2?) that register a customer (from table2) that has purchased parts from table 1. because it could be multiple purchased parts, can these parts be shown in some sort of (pretty) dynamic table/list with all the info from the parts table?

Just need a direction to the right way :)

**Comment by mjashanks** [comment, marked as answer]
Hi There - welcome!

So...
1. ON your customer table, create a new column....
- Type: Relationship
- Name: Parts (?)
- Table: Parts (Table 1)
- Relationship Type: `One Customer has Many Parts`

2. This guide should show you how to display your Parts inside the Customer's record: https://docs.budibase.com/guides/master-detail

Let me know if you need any more help!

**Reply by renrom** [reply]
I think it is indeed just as simple as you are saying! Thanks for the quick reply.
